docs/setup/kubeedge_configure.md

```diff
@@ -0,0 +1,445 @@
+# KubeEdge Configuration
+
+KubeEdge requires configuration on both [Cloud side (KubeEdge Master)](#configuration-cloud-side-kubeedge-master) and [Edge side (KubeEdge Worker Node)](#configuration-edge-side-kubeedge-worker-node)
+
+## Configuration Cloud side (KubeEdge Master)
+
+Setting up cloud side requires two steps
+
+1. Modification of the configuration files
+2. Adding the edge nodes (KubeEdge Worker Node) on the Cloud side (KubeEdge Master).
```

Comment:
Now we have supported that automatically register edge node from edgecore. Ref: https://github.com/kubeedge/kubeedge/blob/master/docs/setup/setup.md#deploy-the-edge-node

Reply:
Node auto registration is now the recommended way for users.